requirements/edx/github.txt

```diff
@@ -78,8 +78,8 @@ git+https://github.com/edx/XBlock.git@xblock-0.4.4#egg=XBlock==0.4.4
 -e git+https://github.com/edx/event-tracking.git@0.2.1#egg=event-tracking==0.2.1
 -e git+https://github.com/edx/django-splash.git@v0.2#egg=django-splash==0.2
 -e git+https://github.com/edx/acid-block.git@e46f9cda8a03e121a00c7e347084d142d22ebfb7#egg=acid-xblock
--e git+https://github.com/edx/edx-ora2.git@0.2.7#egg=ora2==0.2.7
--e git+https://github.com/edx/edx-submissions.git@0.1.3#egg=edx-submissions==0.1.3
+-e git+https://github.com/edx/edx-ora2.git@efischer/clear_submission#egg=ora2
+-e git+https://github.com/edx/edx-submissions.git@efischer/soft_delete_sub#egg=edx-submissions
```

Comment:
These will be updated to proper release version before merging this PR, this was just to get my sandbox set up.